gpiozero/input_devices.py

```diff
@@ -598,6 +617,16 @@ def __init__(
             self.close()
             raise
 
+    @property
+    def value(self):
+        """
+        With the default *queue_len* of 1, this is effectively boolean where 0
+        means no motion detected and 1 means motion detected. If you specify
+        a *queue_len* greater than 1, this will be an averaged value where
+        values closer to 1 imply motion detection.
```

Comment:
Ahhhh, so this explains my confusion at https://github.com/RPi-Distro/python-gpiozero/pull/737#discussion_r255509849 :grinning: 

If it's only "effectively boolean" and not "actually boolean" does that mean the GardenLight example in https://gpiozero.readthedocs.io/en/master/source_values.html#internal-devices **should** be booleanizing the MotionSensor too?